src/services/issuer.rs

```diff
@@ -155,8 +152,8 @@ where
         issuer_id,
         tag: tag.to_owned(),
         value: CredentialDefinitionData {
-            primary: credential_public_key.get_primary_key()?.try_clone()?,
-            revocation: credential_public_key.get_revocation_key()?,
+            primary: credential_public_key.get_primary_key().try_clone()?,
+            revocation: credential_public_key.get_revocation_key().cloned(),
```

Comment:
@andrewwhitehead This is optional in the type definition. Is it sufficient to just use `clone` like this?

Reply:
`cloned` is a method of `Option<&T>`, it converts to `Option<T>` (where T: Clone).